### ISSUE TITLE
Add option to disable notifications

### ISSUE THREAD

**Issue by CromFr** [jangernert/FeedReader#651]
I'm following a lot of RSS feeds and FeedReader is sending notifications every time it syncs, a saying new articles have been downloaded.
As I don't want to read those articles as they come, it would be nice to be able to disable notifications in the options.

**Comment by brendanlong** [comment]
Huh I thought we had an option for this but I guess not. I agree that there should be.

To do this:

 - [ ] Add [a setting](https://github.com/jangernert/FeedReader/blob/master/schemas/org.gnome.feedreader.gschema.xml)
 - [ ] Add it to [the settings dialog](https://github.com/jangernert/FeedReader/blob/master/src/Widgets/SettingsDialog.vala)
 - [ ] Use the config in [Notification.vala](https://github.com/jangernert/FeedReader/blob/master/src/Notification.vala)

I think that's all the steps? I can try to look into this sometime but I haven't had a lot of time to work on this app recently.

**Comment by CromFr** [comment]
@bilelmoussaoui @brendanlong I tried your solution, but it doesn't work:
![image](https://user-images.githubusercontent.com/999188/39047940-11b985da-449c-11e8-8e47-1f05dcd5caee.png)

I got this notification while only the daemon was running (the GUI was closed and did not appear in process list). I suppose notifications are sent by the feedreader daemon, and GNOME only offers to block the GUI application.

**Comment by brendanlong** [comment]
Oh I see. We removed the daemon a while back, so you just need to get onto the latest version. There is a way to run the new version in background mode, but it should appear to GNOME as exactly the same program as the foreground version (because it is).

**Comment by brendanlong** [comment]
I'm tempted to re-open this, since I think having to navigate to a different app to disable notifications is pretty annoying, especially since GNOME's notifications don't offer an easy way to turn them off in the place where they appear (unlike Android).

Is whatever setting GNOME's Settings is using something we could set programatically from FeedReader?

**Comment by bilelmoussaoui** [comment]
Well, Instead of  implementing something here, I'm hoping we can have an easy way to ask GCC (GNOME Control Center) to open this for us
![Screenshot from 2019-06-12 18-21-16](https://user-images.githubusercontent.com/7660997/59368633-f9350100-8d3e-11e9-83db-2ed399a503a9.png)

**Comment by brendanlong** [comment]
Yeah that would work. Another option would be if the notification trade grew a way to disable notifications when they appear, like Android does:

![Screenshot_20190612-122901](https://user-images.githubusercontent.com/1447206/59369180-d76f5600-8d0d-11e9-8f0c-1b1dceb201ad.png)

(You have more interaction with other GNOME people so maybe you'd know who to suggest this to?)

**Comment by geeseven** [comment]
I would like to add instructions on how to disable notifications for  [emersion/mako][0].  Add the following to `~/.config/mako/config`:

```ini
[desktop-entry=org.gnome.FeedReader]
invisible=1
```


[0]:  https://github.com/emersion/mako

**Comment by ploum** [comment]
I confirm that, in Regolith Linux, it's impossible to disable the notifications for FeedReader (I disabled the notifications in the GNOME control center but I still see them)
